src/js/polyfills/fetch/body.js

```diff
@@ -3,7 +3,7 @@ function isDataView(obj) {
 }
 
 function consumed(body) {
-    if (body._noBody) {
+    if (body._bodySize===0) {
```

Comment:
Can you please run the linter? There are a number of style errors in the JS code.